rust/agama-lib/src/product/client.rs

```diff
@@ -123,9 +123,10 @@ impl<'a> ProductClient<'a> {
 
     /// register product
     pub async fn register(&self, code: &str, email: &str) -> Result<(u32, String), ServiceError> {
-        let mut options: HashMap<&str, zbus::zvariant::Value> = HashMap::new();
+        let mut options: HashMap<&str, &zbus::zvariant::Value> = HashMap::new();
+        let value = zbus::zvariant::Value::from(email.to_string());
```

Comment:
how long  the email lives? Str has From for &str - https://docs.rs/zvariant/latest/zvariant/struct.Str.html#impl-From%3C%26str%3E-for-Str%3C'a%3E

Reply:
Yes, we do not need to do the conversion.